docs/using-oci-volume-provisioner.md

```diff
@@ -0,0 +1,173 @@
+# OCI Volume Provisioner
+
+The OCI Volume Provisioner enables [dynamic provisioning][1] of storage
+resources when running Kubernetes on Oracle Cloud Infrastructure (OCI). It uses
+the [OCI Flexvolume Driver][2] to bind storage resources to Kubernetes Nodes.
+The volume provisioner offers support for [Block Volumes][5] and File System
+Storage volumes.
+
+
+## Deployment
+
+The oci-volume-provisioner is provided as a Kubernetes Deployment.
+
+### Prerequisites
+
+  - Install the [OCI flexvolume driver][2] if provisioning Block Storage
+    volumes.
+
+
+### Configuration
+
+You will first need to create a cloud-provider configuration file tailored to
+your OCI account and Kubernetes cluster. Please see the provider configuration
+documentation [here][5].
```

Comment:
This link takes me to OCI docs on block volume which does not tell me what to put into a cloud-provider.yaml

This is maybe supposed to point to `/docs/development.md#Running-locally` ?
If so, consider revising sentence, same issue as **configuration** section in `using-oci-flexvolume-driver.md`